pkg/machine/qemu/machine.go

```diff
@@ -1171,6 +1172,52 @@ func (v *MachineVM) getForwardSocketPath() (string, error) {
 	return filepath.Join(path, "podman.sock"), nil
 }
 
+func (v *MachineVM) setConfigPath() error {
+	vmConfigDir, err := machine.GetConfDir(vmtype)
+	if err != nil {
+		return err
+	}
+
+	configPath, err := NewMachineFile(filepath.Join(vmConfigDir, v.Name)+".json", nil)
+	if err != nil {
+		return err
+	}
+	v.ConfigPath = *configPath
+	return nil
+}
+
+func (v *MachineVM) setReadySocket() error {
+	rtPath, err := getRuntimeDir()
+	if err != nil {
+		return err
+	}
+	virtualSocketPath, err := NewMachineFile(filepath.Join(rtPath, "podman", v.Name+"_ready.sock"), nil)
+	if err != nil {
+		return err
+	}
+	v.ReadySocket = *virtualSocketPath
+	return nil
+}
+
+func (v *MachineVM) setQEMUAndPIDSocket() error {
+	rtPath, err := getRuntimeDir()
+	if err != nil {
+		return err
+	}
+	if !rootless.IsRootless() {
+		rtPath = "/run"
+	}
+	socketDir := filepath.Join(rtPath, "podman")
+	pidFilePath, err := NewMachineFile(filepath.Join(socketDir, fmt.Sprintf("%s.pid", v.Name)), nil)
+	if err != nil {
+		return err
+	}
+	v.PidFilePath = *pidFilePath
+	return nil
+}
+
+// Deprecated: getSocketandPid is being replace by setQEMUAndPIDSocket and
```

Comment:
Why is this being kept, can't it just be deleted?

Reply:
nope, need it for migrating old configuration files